client/modules/Rewards/Rewards.jsx

```diff
@@ -4,6 +4,7 @@ import Helmet from 'react-helmet';
 
 import Header from 'components/Header/Header';
 import Footer from 'components/Footer/Footer';
+import RewardTypes from './components/RewardTypes/RewardTypes';
```

Comment:
You didn't make a folder to contain your `RewardTypes` component + assets, so this file can't be found.